crates/aptos-rest-client/src/lib.rs

```diff
@@ -120,6 +120,27 @@ impl Client {
         })
     }
 
+    pub async fn get_account_balance_at_version(
+        &self,
+        address: AccountAddress,
+        version: u64,
+    ) -> Result<Response<Balance>> {
+        let resp = self
+            .get_account_resource_at_version(
+                address,
+                "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>",
```

Comment:
Not worth doing here but we really ought to use a constant here, this is in our code base like a hundred times.